### ISSUE TITLE
Bug : No Focus Sony XPeria Phones : XPeria M2 and XPeria M4 Aqua

### ISSUE THREAD

**Issue by edynamic90** [Redth/ZXing.Net.Mobile#381]
Hello, 

There is no focus with last version 2.1.12 and 2.1.13 on XPeria M2 and XPeria M4 Aqua (Android 5.1.1. and 6.0)

It's blur, indeed very blur with no scan for result

With version 2.4.0.17, no problem with focus

Best regards

**Comment by F1nZeR** [comment]
Does autofocus works in other camera apps? What about focus on screen touch?

**Comment by edynamic90** [comment]
It works with other camera apps and focus is OK too

No focus with last version 2.1.12 and 2.1.13 **BUT no problem with 2.4.0.17**

I developp on Xamarin 6.0.1 build 9 and API 22

**Comment by edynamic90** [comment]
Hello,

With the last version 2.1.44, no focus, the preview is blurry always

After flash activated and wait 3-5, focus is triggered and the preview is OK

But it's only after flashlight using, it's a bug 

I developp on Xamarin 6.0.2 build 73 (last version) with API 22 and 23

Best regards

**Comment by hmoratopcs** [comment]
Does the camera focus when you touch the screen? Possible duplicate of #395

**Comment by Redth** [comment]
Thanks for reporting this issue!  Unforunately it took me way too long to respond 😭.  Sorry, I apologize!  Recently the source code for this project was completely refactored to modernize it.  Many PR's were included in this effort, and many bugs were hopefully fixed.  Please try out the latest 3.x series of NuGet packages (currently in prerelease).  To try and make the project more maintainable in my spare time going forward, I've decided to close all existing issues to start with a clean slate.  If you're still experiencing this issue on the newest version, please open a new issue with as much detail as possible.  Thank you for your patience and understanding!  Happy scanning!
